requirements_dev.txt

```diff
@@ -14,3 +14,4 @@ ipython
 matplotlib
 properties[math]
 wheel
+pytest
```

Comment:
@rowanc1: thoughts on bringing this in as a dev dependency? It makes testing that warnings are thrown easier. http://doc.pytest.org/en/latest/